Serialization.NET/Segment/Serialization/JsonUtility.cs

```diff
@@ -83,8 +83,8 @@ private static char ToHexChar(int i)
             }
         }
 
-        public static string ToJson(object value, bool pretty = false) => JsonConvert.SerializeObject(value, pretty ? Formatting.Indented : Formatting.None);
+        public static string ToJson(object value) => JsonSerializer.Serialize(value);
```

Comment:
we want to keep `pretty` option

Reply:
Found the new formatting ... format.   Fixed.